### DISCUSSION TITLE
Set up “Interface Aware” Damage model between two different blocks to simulate bonding damage between different materials.

### DISCUSSION THREAD

**Discussion by ErhuiGuo** [peridigm/peridigm#374] [Q&A]
I am performing an impact damage simulation on a polycrystalline material. In order to consider the bonding damage between different crystals, I want to set interface aware damage model between different crystals, but this damage model involves two blocks, under which block should I define it. And is it possible for me to define two damage models under one blocks? One for its own damage and one for bonding damage between it and neighbouring blocks.

**Comment by JTHesse** [comment]
Take a look at [PeriLab.jl](https://github.com/PeriHub/PeriLab.jl). Damage interface models are implemented there.
